python/packages/zarathustra/skills/subgraph_query_abci/skill.yaml

```diff
@@ -154,7 +157,8 @@ models:
     args: {}
     class_name: TendermintDialogues
 name: subgraph_query_abci
-protocols: []
+protocols:
+- valory/ledger_api:1.0.0:bafybeihdk6psr4guxmbcrc26jr2cbgzpd5aljkqvpwo64bvaz7tdti2oni
```

Comment:
ledger api call?

Reply:
i really like the threshold thing to be honest it is a great idea and its nice and simple to put in.

I am a little worried we need to pass 2 sets of configuratiosn to the ledger so that it knows what chain to hit though.

Reply:
yes I have thought about that as well. Not entirely sure tho, might not be the case